spring-data-mock/src/main/java/com/mmnaseri/utils/spring/data/store/impl/AuditDataEventListener.java

```diff
@@ -3,10 +3,11 @@
 import com.mmnaseri.utils.spring.data.domain.RepositoryMetadata;
 import com.mmnaseri.utils.spring.data.store.DataStoreEvent;
 import com.mmnaseri.utils.spring.data.store.DataStoreEventListener;
-import org.joda.time.DateTime;
 import org.springframework.data.domain.Auditable;
 import org.springframework.data.domain.AuditorAware;
 
+import java.time.Instant;
```

Comment:
Does this mean that jodatime can be removed as a dependency?

Reply:
It's already removed :)